templates/layout/base.html

```diff
@@ -3,6 +3,11 @@
     <head>
         <title>{{ head.title }}</title>
         {{{ resourceHints }}}
+
+        {{#if carousel.slides.[0].image}}
```

Comment:
Have you tested this with a store with no carousel? Interested to know what the context looks like in that case and whether this conditional still works.